### ISSUE TITLE
Several issues on this one: missing application name/title, application icon and also cannot close application window.

### ISSUE THREAD

**Issue by ak2766** [antimof/UxPlay#35]
This one has several issues on Ubuntu 20.04.2 LTS fully patched.

Running a locally compiled UxPlay with the following `git log` output:
```
$ git log
commit 6a473d6026480c47b6d9f1b2d619039da3cd36ba (HEAD -> master, origin/master, origin/HEAD)
Author: antimof <antimof@users.noreply.github.com>
Date:   Thu May 7 09:22:39 2020 +0300

    Check availability of necessary Gstreamer plugins
```

Issues noted (see screenshot):
1.  Application is unknown to the OS
2. Application has no name/title - most likely causing issue 1 above
3. Application has no icon
4. Application cannot be closed by clicking the close icon in top top left of application or via ALT+F4

![Selection_01082](https://user-images.githubusercontent.com/6629646/107613747-07b68700-6c9d-11eb-9b7c-f54b5754ca63.png)

**Comment by D3SOX** [comment]
I think #31 fixes the name/title problem

**Comment by ak2766** [comment]
Than @D3SOX.

Just before posting here, I did do a `git pull` but got the message `Already up to date`.  Most likely that change hasn't been merged yet.  I'll keep checking back to see when that's sorted.

**Comment by D3SOX** [comment]
Yes, you might want to merge it locally since I don't think this project is maintained anymore

**Comment by ak2766** [comment]
Unfortunately, in addition to not changing anything on the app display (still the same as the screenshot above), it introduced a delay of ~15 seconds.  That is, if I launch an app on the phone, it takes ~15 seconds before that action is relayed to screen.  This is on an iPhone 6S - not sure if that has any bearing.

But I've had to revert that patch.

**Comment by fduncanh** [comment]
David Venturas patch #31 (the complete patch)   certainly does work (on iPad. at least) and does add a title, unlike your screen shot.   Maybe you (ak2766)  havent applied it right?

Its been invaluable to me for using UxPlay for  teaching with Zoom.

**Comment by fduncanh** [comment]
There is an updated UxPlay pr # 58 which might fix things.   run cmake with -DZOOMFIX=ON as its README.md describes

**Comment by ak2766** [comment]
@fduncanh - You might be onto something there about me incorrectly applying the patch.  I'll test this again soon.
